pkg/types/ccipocr3/interfaces.go

```diff
@@ -53,3 +58,15 @@ type ExecutePluginCodec interface {
 type MessageHasher interface {
 	Hash(context.Context, CCIPMsg) (Bytes32, error)
 }
+
+type HomeChainPoller interface {
```

Comment:
Where is this used?

Reply:
https://github.com/smartcontractkit/ccip/actions/runs/9614422103/job/26519178069?pr=978

Reply:
It will be used in all CCIP plugins to get the dynamic configs from Home Chain